.circleci/config.yml

```diff
@@ -12,11 +12,25 @@ workflows:
         - test-2.071-prod
         - test-2.077-upstream-devel
         - test-2.077-upstream-prod
+        - d2tag:
+            context: org-global
+            requires:
+            - test-2.070-devel
+            - test-2.070-prod
+            - test-2.071-devel
+            - test-2.071-prod
+            - test-2.077-upstream-devel
+            - test-2.077-upstream-prod
+            filters:
+                tags:
+                    ignore: /.*\+d2$/
+                branches:
+                    ignore: /.*/
 
 templates:
     test: &test-template
         docker:
-        - image: sociomantictsunami/dlang:xenial-v3.2.0-rc.1
+        - image: sociomantictsunami/dlang:xenial-v3.2.0
```

Comment:
Why not just `v3` to get automatic updates? Minor versions should be reasonably reliable, but we could also start tagging `v3.2` too if you prefer to only get patch-updates.